lesson18/index.js

```diff
@@ -100,12 +99,10 @@ const createSliderBtns = () => {
 }
 
 const controlBtnBehavior = () => {
-  const allSlides = Array.from(document.querySelectorAll(".sliderImg"));
   const allSlidesLength = Array.from(document.querySelectorAll(".sliderImg")).length;
-  const currentSlide = document.querySelector('[data-hidden="false"]');
   const prevBtn = document.getElementById("js-slider-previousBtn");
   const nextBtn = document.getElementById("js-slider-nextBtn");
-  const currentSlideIndex = allSlides.indexOf(currentSlide);
+  const currentSlideIndex = globalIndex;
```

Comment:
Q. Why assign globalIndex to another variable?

Reply:
[73aca0b](https://github.com/hippopo-moo/js-handson-work/pull/20/commits/73aca0b19b03d8fb6aa4ec909859672e0aa9a64d)

It was an unnecessary substitution. Fixed!